src/components/common/PeopleAndRoles/PeopleAndRoles.vue

```diff
@@ -124,6 +124,62 @@
         </div>
       </article>
 
+      <!-- Restoration conversion and extension add buttons -->
+      <article v-if="(isLimitedConversionRestorationFiling || isLimitedExtendRestorationFiling)"
+        class="section-container">
+        <header>
+          <h2 id="resto-heasder-lbl">Add Applicant Information</h2>
+        </header>
+        <section class="mt-4">
+          <h3>Your application must include one of the following:</h3>
+        </section>
+        <ul>
+          <li>
+            <v-icon>mdi-circle-small</v-icon>
+            <span>An individual</span>
+          </li>
+          <li>
+            <v-icon>mdi-circle-small</v-icon>
+            <span>A business or a corporation</span>
+          </li>
+        </ul>
```

Comment:
Is it possible that adding this code to `PeopleAndRoles` might be mixing concerns?  If this code is relevant to Restoration filings only, can we place it in separate component and call it from the parent?

Reply:
If we do this then we should do it for several such sections/articles, which I think should be in a separate cleanup ticket, not this ticket.